packages/protocol/contracts/rrp/authorizers/DaoRequesterRrpAuthorizer.sol

```diff
@@ -0,0 +1,135 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.6;
+
+import "../../adminnable/Adminnable.sol";
+import "./RequesterRrpAuthorizer.sol";
+
+/// @title Authorizer contract that whitelists requesters where the API3 DAO is
+/// the meta-admin
+/// @dev Unlike the MetaAdminnable contract that it inherits, this contract
+/// does not support the independent adminning of different entities. In other
+/// words, when an address is assigned to be an Admin/SuperAdmin, it becomes
+/// the Admin/SuperAdmin of all entities being adminned. To achieve this, the
+/// interface that allows the caller to specify an `adminnedId` is disabled,
+/// and an alternative one is implemented where `adminnedId` is forced to be
+/// `bytes32(0)`.
+contract DaoRequesterRrpAuthorizer is Adminnable, RequesterRrpAuthorizer {
```

Comment:
It seems we are missing an interface similar to https://github.com/api3dao/airnode/blob/beta-protocol-endpoint-whitelisting/packages/protocol/contracts/rrp/authorizers/interfaces/IAirnodeRequesterRrpAuthorizer.sol#L0-L1

Reply:
Good catch, added https://github.com/api3dao/airnode/pull/556/commits/6317f579409338dd003b7962b11fd276d9730b0c